home/web/client/actions/dapps.js

```diff
@@ -0,0 +1,5 @@
+import { createAction } from 'redux-actions';
+
+export const error = createAction('error dapps');
```

Comment:
following our conventions:
'action context'

this means that there was an error with dapps, like we have in status page:
`export const error = createAction('error rpc')`